src/sentry/api/base.py

```diff
@@ -582,6 +583,25 @@ def paginate(
         self.add_cursor_headers(request, response, cursor_result)
         return response
 
+    def get_appropriate_user_id(self, request: Request) -> int:
```

Comment:
is this method used by other endpoints? if not, maybe we can move it to `api_tokens.py` b/c `base.py` already has a lot of logic that lives there.

Reply:
fixed!